modules/service/src/test/scala/lucuma/odb/graphql/feature/calibrations.scala

```diff
@@ -924,8 +999,8 @@ class calibrations extends OdbSuite with SubscriptionUtils {
                           "value" : null
                         }
                       }
-                    """,
-                  )})).mapN(_ ::: _)
+                    """
+                  })).mapN(_ ::: _)
```

Comment:
I think there may be a minor issue with the test case here.  The order that the events arrive doesn't seem to be guaranteed?  Changes elsewhere in the test case impact the ordering of events for this unrelated program.

Reply:
That's right, order is not guaranteed and tests are a bit ad hoc

Reply:
Ha, I was just looking at that. For operations that generate multiple events via a bulk update we might want to compare sets rather than lists.